src/server/controller/donate.js

```
@@ -51,6 +51,15 @@ async function chargeTransaction(req, res) {
 
   if (chargeObj.paymentMethod === 'stripe') {
     try {
+      // make sure item statuses are all VERIFIED
+      const items = await itemHelpers.getItemObjsFromItemIds(chargeObj.itemIds);
+      if (items.some(item => item.status !== 'VERIFIED')) {
+        const warningMsg = `WARNING - chargeTransaction: one or more items is not VERIFIED: ${chargeObj.itemIds}`;
+        errorHandler.raiseWarning(warningMsg);
+        console.log(warningMsg);
+        return res.status(500);
```

let's either throw an error here, or return a message (b/c the front-end is expecting a message string to render to the user. I.e: `res.status(500).send({message: 'Stripe experienced an error!'})` or something like that. Refer to the catch block below @lucashu1

---

Got it, will do